diff.go

```diff
@@ -53,12 +53,25 @@ const (
 	// DiffInfoComponentExposeUpdated
 	DiffInfoComponentExposeUpdated DiffType = "component-expose-updated"
 
-	// DiffInfoComponentScaleUpdated
-	DiffInfoComponentScaleUpdated DiffType = "component-scale-updated"
+	// scale
+
+	// DiffInfoComponentScalePlacementUpdated
+	DiffInfoComponentScalePlacementUpdated DiffType = "component-scale-placement-updated"
+
+	// DiffInfoComponentScaleUp
+	DiffInfoComponentScaleUp DiffType = "component-scale-up"
```

Comment:
I agree with the naming. Will fix it. 

Reason for not having `component-min-updated` is that we want to distinguish in a more detailed way what exactly was updated in min. So we can evaluate if we want to scale up or scale down.